test/crowdsale/ZxcCrowdsale.test.js

```diff
@@ -744,6 +796,25 @@ contract('crowdsale/ZxcCrowdsale', (accounts) => {
       assert.strictEqual(actualTokens.toString(), expectedTokens.toString());
     });
 
+    it('buyTokens should purchase tokens if buyer kyc token is level 1 in crowdsale', async () => {
```

Comment:
Update existing test for buyerOne to mention KYC level

Reply:
https://github.com/0xcert/ethereum-crowdsale/pull/7/commits/090f7533070cb6977cd8918bd05b17c54b104447